examples/authsignal-passwordless/components/login.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./login.module.css";
+
+export const Login = () => (
+  <main className={styles.login}>
+    <h1 className={styles.title}>My Example App</h1>
+    <form method="POST" action="/api/login">
+      <label htmlFor="email">Email</label>
+      <input id="email" type="email" name="email" required autoFocus />
```

Comment:
```suggestion
      <input id="email" type="email" name="email" required/>
```

Autofocusing inputs is bad for a11y, let's remove it.

https://www.a11yproject.com/checklist/#avoid-using-the-autofocus-attribute